locales/en.json

```diff
@@ -0,0 +1,9 @@
+{
+	"login": {
+		"signIn": "Sign In"
+	},
+	"userInformation": {
```

Comment:
userAuthentication*
Also the tabs seem too big, could you please check that

Reply:
The tabs are auto generated, but I will look into it. it should not be called user authentication because these variables are not limited to authentication. we can be using this information across the platform and so information is the more appropriate term

Reply:
I see your point, we can expand this later to include more information. Yeah lets keep it like this for now, good call